test/integration/scheduler/plugins/plugins_test.go

```diff
@@ -343,12 +343,12 @@ func (pp *PreBindPlugin) PreBind(ctx context.Context, state *framework.CycleStat
 		return nil
 	}
 	pp.podUIDs[pod.UID] = struct{}{}
-	if pp.failPreBind {
-		return framework.NewStatus(framework.Error, fmt.Sprintf("injecting failure for pod %v", pod.Name))
-	}
 	if pp.rejectPreBind {
 		return framework.NewStatus(framework.Unschedulable, fmt.Sprintf("reject pod %v", pod.Name))
 	}
+	if pp.failPreBind {
+		return framework.NewStatus(framework.Error, fmt.Sprintf("injecting failure for pod %v", pod.Name))
+	}
```

Comment:
I understand the intention for this change, but this is not necessary, I am fine with it though.

Reply:
Revert.